packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -100,9 +103,7 @@ const TradersHub = () => {
                                 onChange={platformTypeChange}
                                 value={selected_platform_type}
                             />
-                        ) : (
-                            <ButtonToggleLoader />
-                        )}
+                        ) : null}
                         {selected_platform_type === 'options' && <OptionsAndMultipliersListing />}
                         {selected_platform_type === 'cfd' && <CFDsListing />}
```

Comment:
isMT5Allowed(landing_company)
```suggestion
                        {selected_platform_type === 'cfd' && isMT5Allowed(landing_company) && <CFDsListing />}
```

Reply:
Commit suggestion 🙏🏼